apps/playground/app/test-data-list/page.tsx

```diff
@@ -164,13 +164,27 @@ export default function DataListPage() {
                     <DataListItem align="center">
                       <DataListLabel>Authentication methods</DataListLabel>
                       <DataListValue>
-                        <Flex gap="2">
+                        <Flex gap="2" align="center">
                           <StarFilledIcon />
                           <StarFilledIcon />
                         </Flex>
```

Comment:
This yielded different alignment depending on whether text was present within the Flex

Would rather have this as a tradeoff, as the other one was harder to stronghand